filebeat/filebeat.reference.yml

```diff
@@ -861,6 +861,10 @@ filebeat.inputs:
   # if the fingerprints match, then the files are considered equal.
   #file_identity.fingerprint: ~
 
+  # Include file owner name and group name in the event metadata.
+  #include_file_owner_name: true
+  #include_file_owner_group_name: true
```

Comment:
A couple of things here:
1. in the reference documentation we usually use the default values, so those should be `false`
2. This file is auto-generated based on [`filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl`](https://github.com/elastic/beats/blob/main/filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl).

To generate the auto-generated files:
```
cd filebeat
mage check
cd ../x-pack/filebeat
mage check
```

`mage check` does more than just generating the files, it:
- Formats the code
- Generate all files
- Checks whether there are uncommitted changes.

You need to run this on both `filebeat` and `x-pack/filebeat` as the latter uses the config templates from former.